src/robusta/core/sinks/robusta/robusta_sink.py

```diff
@@ -467,6 +474,24 @@ def __should_run_history(self) -> bool:
             logging.error("Failed to check run history condition", exc_info=True)
             return False
 
+    def __discovery_watchdog(self):
+        logging.info("Cluster discovery watchdog initialized")
+        is_initial_health_check = True
+        while self.__active:
```

Comment:
Can we simplify it by initializing the `last_send_time` to current time, inside `__update_cluster_status`?
This way you only need this:
```
            if not self.is_healthy():
                logging.info("Unhealthy discovery, restarting runner")
                # sys.exit and thread.interrupt_main doest stop robusta
                os.kill(os.getpid(), signal.SIGINT)
                return
```

Reply:
its set before __update_cluster_status,
this is more for the rare case that the thread gets stuck before there, like in has_cluster_findings